src/api-endpoints.ts

```diff
@@ -0,0 +1,18 @@
+import { ISdkParams } from './http-client';
+
+const UFC_ENDPOINT = '/flag-config/v1/config';
+
+/** Utility class for constructing an Eppo API endpoint URL given a provided baseUrl and query parameters */
+export default class ApiEndpoints {
```

Comment:
Or consider a rename to `ConfigEndpoint`? I imagine we would use a different class if we need APIs for non-config purposes in the future

Reply:
I made it plural because I see this adding support for other endpoints in the future, not just UFC. I don't feel strongly about it, it's a naming convention that I've seen before and like, but I don't feel strongly about it